common/Unit.cpp

```diff
@@ -24,6 +24,8 @@
 #include <common/SigUtil.hpp>
 
 UnitBase *UnitBase::Global = nullptr;
+UnitKit *GlobalKit = nullptr;
+UnitWSD *GlobalWSD = nullptr;
```

Comment:
If we have two separate variables for each type, why do we need the `UnitBase::Global`? Isn't it better to replace `UnitBase::Global` with `UnitBase::GlobalKit` and `UnitBase::GlobalWSD` ?

Reply:
we need UnitBase::Global for UnitBase::get() which we use for example in WebSocketHandler.cpp where we don't know which kind of test we run (it returns the last set UnitBase subclass)